src/fidesops/models/policy.py

```diff
@@ -70,6 +85,27 @@ class is referenced in multiple database migrations. This class is to be removed
 """
 
 
+def _validate_drp_action(
+    drp_action: Optional[str], drp_action_exists: bool, class_name: str
+) -> None:
+    """Check that DRP action is supported"""
+    if not drp_action:
+        return
+    if drp_action_exists:
```

Comment:
mostly just wanted to map that err case to the `DrpActionValidationError` class so that it's handled the same way as other invalid DRP actions. I can remove and just use the base DB exception, if you think that's better?

Reply:
I think that's better in this case as it makes the code more readable, and saves us two database interactions.